s3_file_uploads/utils.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+from rest_framework.exceptions import ValidationError
+
+def validate_acl_type(data):
+    if data and data.get('acl') not in settings.ACCESS_CONTROL_TYPES:
+        raise ValidationError('{acl} is not a valid acl type'.format(acl=data.get('acl')))
+    return data
+
+
+def get_file_data_from_request_data(request_data):
+    return {
+        k: v
+        for k, v
+        in request_data.items()
+        if k != 'acl'
+    }
+
+def get_acl_data_from_request_data(request_data):
+    return {
+        k: v
+        for k, v
+        in request_data.items()
+        if k == 'acl'
```

Comment:
could you rename the variables in these functions to be explicitly `key` and `value` as we normally avoid single letter variable names.

Reply:
Additionally could you put `'acl'` into a meaningful constant that is used in all the places it's used please? :)

Reply:
Sure - added as a constant in the settings